source/common/config/new_grpc_mux_impl.h

```diff
@@ -9,52 +9,62 @@
 #include "common/config/delta_subscription_state.h"
```

Comment:
@wgallagher I assume eventually we are going to rename files away from "new" if we only have a single implementation? Not sure of the history here.

Reply:
Yeah, totally. This PR keeps it as new to minimize churn, and most importantly to allow the github diffs in this PR to actually be meaningful for those files.